sotodlib/preprocess/processes.py

```diff
@@ -819,18 +825,18 @@ class AzSS(_Preprocess):
         calc:
           signal: 'demodQ'
           azss_stats_name: 'azss_statsQ'
-          range: [-1.57079, 7.85398]
+          azrange: [-1.57079, 7.85398]
```

Comment:
Why isn't this called "az_range"?  I suspect it's "too late now" but please be careful in the future to be consistent in things like word separation.